components/molecule/field/src/index.scss

```diff
@@ -24,7 +24,6 @@ $atomHelpText-class: '.sui-AtomHelpText';
 
       #{$atomLabel-class} {
         align-items: center;
-        margin-left: $ml-molecule-field-label;
```

Comment:
wouldn't this break somewhere?

Reply:
Not it won't in order to both label and nodelabel to have the same behavior a div wrapper have been added to add this padding to both cases. This have been done in order to ensure retrocompatibility